pkg/services/folder/folderimpl/folder.go

```diff
@@ -178,7 +179,45 @@ func (s *Service) GetChildren(ctx context.Context, cmd *folder.GetChildrenQuery)
 			continue
 		}
 
-		g, err := guardian.NewByUID(ctx, f.UID, f.OrgID, cmd.SignedInUser)
+		g, err := guardian.NewByUID(ctx, f.UID, f.OrgID, q.SignedInUser)
+		if err != nil {
+			return nil, err
```

Comment:
Can you point me to the exact function: I think that we [fail](https://github.com/grafana/grafana/blob/f8cf67347f069310b8cc6a2d9ff83345a81e3247/pkg/services/folder/folderimpl/folder.go#L132) also the `Get()` in that case

Reply:
Sure, this is the part I meant:
https://github.com/grafana/grafana/blob/29c6dc5e64d16060c7d49d5d0b64b9ae126584f7/pkg/services/folder/folderimpl/folder.go#L176-L180

Reply:
I see, I have changed both `GetChildren()` and `GetDescendants()` so as they continue in case of errors.
(I have also changed the error logs to debug logs to reduce the noise)